### PR TITLE
Readthedocs update

### PR DESCRIPTION
**Pull request by ferrigno**
Just adding how to retrieve a token stored locally and show the information on token wit logging.

### CODE REVIEW

File: doc/source/user_guide/Authentication.ipynb [hunk 1]
```diff
@@ -72,17 +104,63 @@
...
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "06dbd397",
+   "metadata": {},
+   "source": [
+    "# Storing your token locally\n",
+    "\n",
+    "The token cn be sotred in \n",
```
Comment (volodymyrss) [inline doc/source/user_guide/Authentication.ipynb:114]:
```suggestion
    "The token can be stored in \n",
```

File: doc/source/user_guide/Authentication.ipynb [hunk 2]
```diff
@@ -72,17 +104,63 @@
...
+    "- an environment variable called ODA_TOKEN\n",
+    "- a file called ```.oda-token``` in current directory\n",
+    "- a file called ```.oda-token``` in your home diretory \n",
```
Comment (volodymyrss) [inline doc/source/user_guide/Authentication.ipynb:117]:
```suggestion
    "* an environment variable called ODA_TOKEN\n",
    "* a file called `.oda-token` in current directory\n",
    "* a file called `.oda-token` in your home diretory \n",
```

**Review by volodymyrss — APPROVED** [review]
the markdown list was not showing correctly, it needed new line for some reason.
Seems ok now but I am not sure to read all.

**Comment by ferrigno** [comment]
Let's merge and see how it displays.

**Comment by volodymyrss** [comment]
> Let's merge and see how it displays.

you can see how it displays before merge, here: https://oda-api--80.org.readthedocs.build/en/80/
this is how I knew it does not display list

**Pull request merged**